infrastructure/key-vault.tf

```diff
@@ -8,16 +8,17 @@ data "azurerm_key_vault" "s2s_vault" {
   resource_group_name = "rpe-service-auth-provider-${var.env}"
 }
 
-#data "azurerm_key_vault_secret" "civil_sdt_service_s2s_key" {
-#  name         = "microservicekey-civil-sdt-service"
-#  key_vault_id = data.azurerm_key_vault.s2s_vault.id
-#}
-#
-#resource "azurerm_key_vault_secret" "civil_sdt_service_s2s_secret" {
-#  name         = "civil-sdt-service-s2s-secret"
-#  value        = data.azurerm_key_vault_secret.civil_sdt_service_s2s_key.value
-#  key_vault_id = data.azurerm_key_vault.civil_vault.id
-#}
+
+data "azurerm_key_vault_secret" "key_from_vault" {
+  name         = "microservicekey-civil-sdt"
+  key_vault_id = data.azurerm_key_vault.s2s_vault.id
+}
+
+resource "azurerm_key_vault_secret" "s2s" {
+  name         = "civil-sdt-service-s2s-secret"
+  value        = data.azurerm_key_vault_secret.key_from_vault.value
+  key_vault_id = data.azurerm_key_vault.civil_vault.id
```

Comment:
Key vault has changed to civil-sdt specific vault.  Update this branch with latest changes from main branch and change key_vault_id value to `module.civil_sdt_key_vault.key_vault_id`